mitosheet/src/jupyter/api.tsx

```diff
@@ -721,7 +721,24 @@ export default class MitoAPI {
         }, {})
     }
 
+    
+    async editTranspose(
```

Comment:
Instead of creating an editTranspose separate function call, do we want to just use the standard `_edit` api call?

Reply:
We only do this when we never call it anywhere else currently, so leaving.